ticketreminder/api.py

```diff
@@ -421,22 +426,32 @@ def _do_send(self, id, ticket, author, origin, description):
                 """, (id,))
 
 
-class TicketReminderNotifyEmail(TicketNotifyEmail):
-    def __init__(self, env, reminder):
-        super(TicketReminderNotifyEmail, self).__init__(env)
-        self.reminder = reminder
-
-    def _notify(self, ticket, newticket=True, modtime=None):
-        description = ticket.values.get('description')
-        ticket.values['description'] = self.reminder
-        super(TicketReminderNotifyEmail, self)._notify(ticket, newticket, modtime)
-        ticket.values['description'] = description
-
-    def notify(self, ticket):
-        super(TicketReminderNotifyEmail, self).notify(ticket, newticket=True)
-
-    def format_subj(self, summary, newticket=True):
-        return super(TicketReminderNotifyEmail, self).format_subj("Ticket reminder", newticket)
+#class TicketReminderNotifyEmail(TicketNotifyEmail):
+#    def __init__(self, env, reminder):
+#        super(TicketReminderNotifyEmail, self).__init__(env)
+#        self.reminder = reminder
+#
+#    def _notify(self, ticket, newticket=True, modtime=None):
+#        description = ticket.values.get('description')
+#        ticket.values['description'] = self.reminder
+#        super(TicketReminderNotifyEmail, self)._notify(ticket, newticket, modtime)
+#        ticket.values['description'] = description
+#
+#    def notify(self, ticket):
+#        super(TicketReminderNotifyEmail, self).notify(ticket, newticket=True)
+#
+#    def format_subj(self, summary, newticket=True):
+#        return super(TicketReminderNotifyEmail, self).format_subj("Ticket reminder", newticket)
+
+class TicketReminderEvent(NotificationEvent):
+    """Represent a ticket reminder `NotificationEvent`."""
+
+    def __init__(self, target, time, author, comment=None):
+        
+        super(TicketChangeEvent, self).__init__('ticket', 'reminder', target,
+	                                                time, author)
```

Comment:
This class may not be necessary. `TicketChangeEvent` has a 6th arg `comment`. Can `TicketChangeEvent` be used instead?